config.go

```diff
@@ -31,10 +31,10 @@ var (
 	BuildRevision = ""
 
 	// Commandline Version String; used to control auto-updating.
-	ClientVersion = "2020-09-29"
+	ClientVersion = "2020-09-30"
 
 	// Agent version to control agent rollover.
-	AgentVersion = "2020-09-21"
+	AgentVersion = "2020-09-30"
```

Comment:
https://github.com/evergreen-ci/evergreen/commit/ac8229973df2a24ed1291b48517aff4d5d82b723 should have incremented the agent version but didn't, so I'm slipping it in here